recipe/meta.yaml

```diff
@@ -75,11 +74,9 @@ outputs:
         - tbb-devel
         - mppp >=0.27
         - fmt
-        - boost-cpp
```

Comment:
This removal is almost certainly incorrectly - it happened because the logic of the bot is not smart enough for all corner cases (and the match with `boost-cpp` in the run-exports messed up the expected counts.